test/e2e/test_e2e.py

```diff
@@ -25,7 +25,8 @@ def test_e2e(fname):
     dname = path.dirname(__file__)
     root = path.join(dname, 'input')
     model = '/' + fname
-    out_fname = path.join(dname, 'actual_output', fname + '.txt')
+    out_fname = path.join(dname, '..', '..', 'tmp',
+                          'test', 'e2e', 'actual_output', fname + '.txt')
```

Comment:
This is rather fragile. It will fail if this file is moved in the repository and this path is not adjusted. Perhaps it needs a TODO to come up with better tmpfile handling?

Reply:
My original suggestion was to put things in a temp directory (/tmp on *nix). <repo>/tmp was a compromise to keep everything in one place, but still close to the project.

It wasn't a big deal for me, so if it's going to cause more trouble than it's worth, I'm entirely happy for us to drop this PR leave things as they are.

Reply:
The right way to do this would be to split dname into the repository root and subpath, and then insert "tmp" in the middle. That also removes the duplication of "test/e2e" as well. But there's no canonical way to identify the repository root. If this was something that needed to be done in many places, it would be worth doing right, but if n==1 then it can wait for the poor sucker who implements n>1 to do.

Reply:
Right you are. I've updated this file as good as I could.